### PR TITLE
Remote desktop test

### LINKED ISSUE #204: New Remote Desktop
The current `kubeflow-containers-desktop` is a bit unwieldy to maintain and there are a few features that are outside the _actual_ remote desktop that may be better served by other images. 

This includes Jupyter which could be served by jupyterlab images, as well as a browser version of vscode which can also be found in our jupyterlab images. (Note that jupyter is not in the 'remote desktop' part, but vscode is) 

~Additionally, the current remote desktop is proving difficult to internationalize to French completely.~ Done via translation files

**New Candidate image**
Courtesy of Blair, can base new Dockerfile off of https://hub.docker.com/r/consol/ubuntu-xfce-vnc/ with the project at https://github.com/ConSol/docker-headless-vnc-container looking at `Dockerfile.ubuntu.xfce.vnc ` for inspiration. 

**TODO**
Make a basic remote desktop using docker headless and place it here. Will need to do investigation on getting jupyter proxy to get a vnc session up and running.  

**Important / to be researched**
~The typical way of adding a locale to the environment does not play well with remote desktop under ubuntu:18.04. It works just fine (as in the applications like Thunar are also localized) with ubuntu:16.04~
Can use Ubuntu 20.04, see https://github.com/ConSol/docker-headless-vnc-container/issues/96#issuecomment-687112199 

Discussion held [here](https://github.com/StatCan/kubeflow-containers/issues/194)

Do we want all these random packages installed by the current remote desktop?
https://github.com/StatCan/kubeflow-containers-desktop/blob/master/base/Dockerfile#L70 
Obvious keepers are curl, the compression libs, git, curl, wget

### PR DESCRIPTION
**Pull request by Ito-Matsuda**
READY. Will close #204 when completed.
This PR leverages work done in https://github.com/yuvipanda/jupyter-desktop-server (but off in a fork [here](https://github.com/Ito-Matsuda/jupyter-desktop-server) to have jupyter run in the background. 

This contains applications from the original remote desktop over in kubeflow-containers-desktop. 
QGIS (thanks Blair), Rstudio, vscode, libcalc, and firefox are included. 

An installation of miniconda is provided as well. 

As it stands this image is currently 10.2 GB

**Also in this PR**
- install the vs-code-french installation for other images by default as well. 

Rocker rebase done here https://github.com/StatCan/kubeflow-containers/pull/225

### CODE REVIEW

**Comment by Ito-Matsuda** [comment]
Still need to update README, but for now solving merge conflicts with master

File: output/RStudio/start-custom.sh [hunk 1]
```diff
@@ -59,13 +59,12 @@ fi
...
-                 --ServerApp.password='' \
-                 --ServerApp.allow_origin='*' \
-                 --ServerApp.authenticate_prometheus=False \
-                 --ServerApp.base_url=${NB_PREFIX} \
-                 --ServerApp.default_url=${DEFAULT_JUPYTER_URL:-/tree}
+                 --NotebookApp.token='' \
+                 --NotebookApp.password='' \
+                 --NotebookApp.allow_origin='*' \
+                 --NotebookApp.base_url=${NB_PREFIX} \
+                 --NotebookApp.default_url=${DEFAULT_JUPYTER_URL:-/tree}
```
Comment (blairdrummond) [inline output/RStudio/start-custom.sh:70]: I think this not intended?

**Review by blairdrummond — APPROVED** [review]
LGTM! Let's go from here

**Pull request merged**